NitroxClient/Communication/NetworkingLayer/LiteNetLib/LiteNetLibClient.cs

```diff
@@ -18,14 +17,14 @@ public class LiteNetLibClient : IClient
         private readonly NetPacketProcessor netPacketProcessor = new NetPacketProcessor();
         private readonly AutoResetEvent connectedEvent = new AutoResetEvent(false);
         private readonly PacketReceiver packetReceiver;
-        private readonly NetworkDebugger networkDebugger;
+        //private readonly NetworkDebugger networkDebugger;
 
         private NetManager client;
 
-        public LiteNetLibClient(PacketReceiver packetReceiver, NetworkDebugger networkDebugger)
+        public LiteNetLibClient(PacketReceiver packetReceiver/*, NetworkDebugger networkDebugger*/)
```

Comment:
Can this not just be removed?
```suggestion
        public LiteNetLibClient(PacketReceiver packetReceiver)
```